### PR TITLE
Add __version__ attribute 

### LINKED ISSUE #81: Add package version to photosynthesis_metrics.__version__
It will be great to have this commonly used attribute to identify the package version

### PR DESCRIPTION
**Pull request by snk4tr**
Closes #81 

## Proposed Changes

  - Add the commonly used attribute to identify the package version

Better be merged after #83

### CODE REVIEW

**Comment by snk4tr** [comment]
@denproc let's merge this if there are no additional comments

File: photosynthesis_metrics/__init__.py [hunk 1]
```diff
@@ -8,3 +8,6 @@
 from .isc import IS, inception_score
 from .vif import VIFLoss, vif_p
 from .brisque import BRISQUELoss, brisque
+
+
+__version__ = "0.4.0"
```
Comment (denproc) [inline photosynthesis_metrics/__init__.py:13]: According to [PEP8](https://www.python.org/dev/peps/pep-0008/#module-level-dunder-names) the `__version__` should be defined before import statements.

File: setup.py [hunk 2]
```diff
@@ -1,14 +1,33 @@
+import os
 import setuptools
```
Comment (denproc) [inline setup.py:3]: I would suggest to use `from photosynthesis_metrics import __version__` as we do not use additional files with metadata.

File: photosynthesis_metrics/__init__.py (hunk 1 above)
Comment (snk4tr) [reply photosynthesis_metrics/__init__.py:13]: Hmm it means that [even pip does it the wrong way](https://github.com/pypa/pip/blob/master/src/pip/__init__.py) 😄
Ok, I will change that

File: setup.py (hunk 2 above)
Comment (snk4tr) [reply setup.py:3]: Discussed with @denproc offline and decided to leave the existing implementation since it is more conventional and robust one

**Comment by snk4tr** [comment]
@denproc ready

**Pull request merged**